src/AnsiEscapeCodeHandler.h

```diff
@@ -30,11 +30,21 @@ namespace Utils {
 class FormattedText {
 public:
     FormattedText() { }
-    FormattedText(const FormattedText &other) : text(other.text), format(other.format) { }
+
+    FormattedText(const FormattedText &other) {
```

Comment:
} on newline please, to follow the project code style

Reply:
Sorry about that.